test_opensearchpy/test_server/test_plugins.py

```diff
@@ -181,3 +182,107 @@ def test_run_monitor(self):
         self.assertIn("monitor_name", response)
         self.assertIn("period_start", response)
         self.assertIn("period_end", response)
+
+
+class TestIndexManagementPlugin(OpenSearchTestCase):
```

Comment:
We should break tests for each plugin in their own file(s), this is going to get really big.